test/test_mdlp.py

```diff
@@ -195,5 +187,4 @@ def test_mdlp_iris():
         [ 2.,  1.,  2.,  2.],
         [ 2.,  0.,  2.,  2.]]
```

Comment:
I'm genuinely surprised that this didn't change the outputs.

That being said, I'm not a big fan of this unit test, because the dataset is so big. It would be nice to refactor this into a smaller unit test in the future.

Reply:
Ya I was also surprised.  On the other hand I do not have a deep intuition on what exactly the mdlp check is testing so it is hard to be very surprised at how modifying the formula would affect the cuts accepted in this specific case. In general it seems quite likely that it would actually affect the outcome in some cases.